modules/calendar/calendar.php

```diff
@@ -1290,6 +1295,13 @@ class="<?php echo esc_attr(implode(' ', $post_classes)); ?>"
                             <?php $title = esc_html(_draft_or_post_title($post->ID)); ?>
 
                             <span class="item-headline post-title" title="<?php echo esc_attr($title); ?>">
+                                <time
+                                    class="item-headline-time"
+                                    datetime="<?php echo $post_publish_datetime; ?>"
+                                    title="<?php echo date_i18n($default_date_time_format, $post_publish_date_timestamp); ?>"
+                                >
+                                    <?php echo date_i18n('ga', $post_publish_date_timestamp); ?>
```

Comment:
Is the 'ga' string used in multiple places? If so, I think it would make sense to have a constant in the calendar class, named like "DEFAULT_TIME_FORMAT"

Reply:
At this point, no. But it does make sense to move them all to consts for consistency.